### PR TITLE
chore(docs): render with latest docsonnet/crdsonnet

### PR DESCRIPTION
**Pull request by Duologic**
title says it, functional no-op

From the comments below:
> _Some changes_ is the result of a fix in crdsonnet, it now recursively looks for the correct type, before if it didn't find a type attribute it assumed it was a string, now it looks further and defaulting to a string as a real last resort. Related PR: https://github.com/crdsonnet/crdsonnet/pull/16

### CODE REVIEW

File: docs/API/panel/table/index.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ fieldConfig.defaults.custom.withCellOptionsMixin(value)
 
 PARAMETERS:
 
-* **value** (`string`)
+* **value** (`object`)
```
Comment (malcolmholmes) [inline docs/API/panel/table/index.md:234]: If this is supposed to be no-op, how come we have a meaningful change here?

File: docs/API/panel/timeSeries/index.md [hunk 2]
```diff
@@ -495,7 +495,7 @@ fieldConfig.defaults.custom.withSpanNulls(value)
 
 PARAMETERS:
 
-* **value** (`string`)
+* **value** (`boolean`,`number`)
```
Comment (malcolmholmes) [inline docs/API/panel/timeSeries/index.md:499]: This again seems like a meaningful change, not just a docs re-render.

File: docs/API/panel/timeSeries/index.md [hunk 3]
```diff
@@ -508,7 +508,7 @@ fieldConfig.defaults.custom.withSpanNullsMixin(value)
...
```
Comment (malcolmholmes) [inline docs/API/panel/timeSeries/index.md:511]: ditto

File: docs/API/panel/table/index.md (hunk 1 above)
Comment (Duologic) [reply docs/API/panel/table/index.md:234]: This is the result of a fix in crdsonnet, it now recursively looks for the correct `type`, before if it didn't find a `type` attribute it assumed it was a string, now it looks further and defaulting to a string as a real last resort. Related PR: https://github.com/crdsonnet/crdsonnet/pull/16/

File: docs/API/panel/timeSeries/index.md (hunk 2 above)
Comment (Duologic) [reply docs/API/panel/timeSeries/index.md:499]: same as above

File: docs/API/panel/table/index.md (hunk 1 above)
Comment (malcolmholmes) [reply docs/API/panel/table/index.md:234]: :+1: Okay, so edit the description then I can approve :-)

**Review by malcolmholmes — APPROVED** [review]

**Pull request merged**